2025-03-23 22:03:06,422 [DEBUG] [__init__.py:341] matplotlib data path: /home/p0wden/miniconda3/envs/mlopt/lib/python3.9/site-packages/matplotlib/mpl-data
2025-03-23 22:03:06,424 [DEBUG] [__init__.py:341] CONFIGDIR=/home/p0wden/.config/matplotlib
2025-03-23 22:03:06,425 [DEBUG] [__init__.py:1509] interactive is False
2025-03-23 22:03:06,425 [DEBUG] [__init__.py:1510] platform is linux
2025-03-23 22:03:06,448 [DEBUG] [__init__.py:341] CACHEDIR=/home/p0wden/.cache/matplotlib
2025-03-23 22:03:06,449 [DEBUG] [font_manager.py:1580] Using fontManager instance from /home/p0wden/.cache/matplotlib/fontlist-v390.json
